packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -322,10 +322,11 @@ export type TSelectInputProps = {
 
 const defaultProps: Pick<
   TSelectInputProps,
-  'maxMenuHeight' | 'menuPortalZIndex'
+  'maxMenuHeight' | 'menuPortalZIndex' | 'controlShouldRenderValue'
 > = {
   maxMenuHeight: 220,
   menuPortalZIndex: 1,
+  controlShouldRenderValue: true,
```

Comment:
I'm not sure if you'll like this change. This is [by default done by `react-select` under the hood](https://github.com/JedWatson/react-select/blob/1bbbc47f5bdd3bd81f6b95ada5554cdb8578ced5/packages/react-select/src/Select.tsx#L289), so I thought we might mimic that here as well and pass the prop to the `<Select>` component and to the `createSelectStyles` helper function.

This was the easier way of fixing this bug. Another I think would require adding a few more cases here:

https://github.com/commercetools/ui-kit/blob/ed3f0c1985b590ecdaf2b248edd8cb6db4252f6a/packages/components/inputs/select-utils/src/create-select-styles.ts#L255-L259

Reply:
Yes, I believe it makes sense 👍